examples/basic/src/VideoPlayer.tsx

```diff
@@ -431,9 +431,9 @@ class VideoPlayer extends Component {
   toggleDecoration() {
     this.setState({decoration: !this.state.decoration});
     if (this.state.decoration) {
```

Comment:
this.video?.setFullScreen(!this.state.decoration);
would be cleaner! The goal is to remove lines here :)